### DISCUSSION TITLE
new plugin: theme-customizer

### DISCUSSION THREAD

**Discussion by rejetto** [rejetto/hfs#810] [Show and tell]
i just started this new plugin that's supposed to make very basic customizations to the look.
just an enabler for people who don't want to learn css for such basic needs.

<img width="404" alt="image" src="https://github.com/user-attachments/assets/190603e4-aec0-47ba-9a64-c4949bfac7fd">

**Comment by PH5HP** [comment]
Hi,

Nice one. What is the status of the Monitoring Screen page regarding extra data there?

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

**Reply by rejetto** [reply]
#763
